cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -90,6 +90,9 @@ impl ABIMachineSpec for X64ABIMachineSpec {
         64
     }
 
+    /// Required stack alignment in bytes.
+    const STACK_ALIGN: u32 = 16;
```

Comment:
I think this should be a regular function accepting a `CallConv` argument.

Reply:
Makes sense to me.  I've updated the patch accordingly.